how-to/get-started/tispark.md

```diff
@@ -2,7 +2,7 @@
 title: TiSpark Quick Start Guide
 summary: Learn how to use TiSpark quickly.
 category: how-to
-aliases: ['/docs/dev/how-to/deploy/tispark/']
+aliases: ['/docs/dev/how-to/deploy/tispark/','/docs/tispark/tispark-quick-start-guide/']
```

Comment:
Is this newly added alias from a previous link? Could you help provide some info?

Reply:
Alias removed.